converter/converter.go

```diff
@@ -0,0 +1,280 @@
+package converter
+
```

Comment:
Could you add a short description of the converter package here so it shows up in godoc?